Client/Helpers/FxALoginHelper.swift

```diff
@@ -90,8 +96,6 @@ class FxALoginHelper {
         // By the time we reach here, we haven't registered for APNS
```

Comment:
Review the above logic. I don't know if this is correct. I also think it is perfectly fine to ask every time the user logs in manually. Even if they did said no in the past.